### PR TITLE
Remove default command prefix so PowerShell can auto-load the Carbon.Cryptography module

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Carbon.Cryptography/Carbon.Cryptography.psd1 [hunk 1]
```diff
@@ -17,7 +17,7 @@
     RootModule = 'Carbon.Cryptography.psm1'
 
     # Version number of this module.
-    ModuleVersion = '2.3.0'
+    ModuleVersion = '3.0.0'
```
Comment (pnstickne) [inline Carbon.Cryptography/Carbon.Cryptography.psd1:20]: I'm assuming version 2.x line should be not used?

**Review by DecoyJoe — APPROVED** [review]

File: Carbon.Cryptography/Carbon.Cryptography.psd1 [hunk 2]
```diff
@@ -110,7 +110,7 @@
     # HelpInfoURI = ''
 
     # Default prefix for commands exported from this module. Override the default prefix using Import-Module -Prefix.
-    DefaultCommandPrefix = 'C'
+    # DefaultCommandPrefix = ''
```
[the remark this answers is not in the document]
Comment (splatteredbits) [reply Carbon.Cryptography/Carbon.Cryptography.psd1:113]: Module manifests are auto-generated by a PS cmdlet. I've always just left the comments in. No reason, though.

File: appveyor.yml [hunk 3]
```diff
@@ -51,35 +51,11 @@ for:
...
-      # Waiting for job to complete (with `Receive-Job -Wait`) hangs sometimes on Ubuntu
-      while( -not (Get-Module -Name 'Prism' -ListAvailable) )
-      {
-        Start-Sleep -Milliseconds 100
-      }
-
-      .\build.ps1
+  - ps: .\build.ps1
 
 # Build in PowerShell
```
[the remark this answers is not in the document]
Comment (splatteredbits) [reply appveyor.yml:56]: The "official" names are "Windows PowerShell" and "PowerShell". The "Core" part was added when "Windows PowerShell" was "officially" "PowerShell".

File: Carbon.Cryptography/Carbon.Cryptography.psd1 (hunk 1 above)
Comment (splatteredbits) [reply Carbon.Cryptography/Carbon.Cryptography.psd1:20]: Yes. I changed all the command names, an backwards-incompatible change, so I had to rev the version to 3.0.

**Pull request merged**